tests/e2e/kfddistribution/furyctl-2-migrate-from-tempo-to-none.yaml

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: kfd.sighup.io/v1alpha2
+kind: KFDDistribution
+metadata:
+  name: sighup
+spec:
+  distributionVersion: v1.27.2
+  # This section describes how the KFD distribution will be installed
+  distribution:
+    kubeconfig: "/shared/kube/kubeconfig-127"
+    #kubeconfig: "/Volumes/sighup_vol/github.com/sighupio/fury-distribution/kind-local-kubeconfig"
+    #kubeconfig: "{env://KUBECONFIG}" TODO to be fixed
+    # This common configuration will be applied to all the packages that will be installed in the cluster
+    common: {}
+    # This section contains all the configurations for all the KFD core modules
+    modules:
+      networking: 
+        type: calico
+      # This section contains all the configurations for the ingress module
+      ingress:
+        baseDomain: fury.sighup.io
```

Comment:
maybe we should use sighup.cc for testing?

Reply:
yes, good idea